Scripts/Python/QueryGroupManagementOmeREDFISH.py

```diff
@@ -0,0 +1,706 @@
+parser.add_argument('--create', help='Create OME query group, pass in \"y\"', required=False)
+parser.add_argument('--update', help='Update existing OME query group, pass in the group ID. You must also use name, description, fid, oid and value arguments. You can change one or all of these arguments but you must make sure you pass in all these arguments even if you didn\'t change the value, still pass in current value', required=False)
+parser.add_argument('--name', help='Create OME query group, pass in an unique string name', required=False)
+parser.add_argument('--description', help='Create OME query group, pass in an unique description string', required=False)
+parser.add_argument('--fid', help='Create OME query group filter, pass in field ID integer value. Field ID is the type of entry you want to filter for like Service Tag. If creating multiple filters, pass in value using comma separator', required=False)
+parser.add_argument('--oid', help='Create OME query group filter, pass in operator ID integer value. Operator ID is what operator you want to use like > or ==. If creating multiple filters, pass in value using comma separator', required=False)
+parser.add_argument('--loid', help='Create OME query group filter, pass in logical operator ID integer value. Logical operator is used when setting multiple filters. Supported values are \"1\" for AND or \"2\" for OR. If only setting one filter for query group, pass in a value of 0 and if setting multiple filters for query group, the first filter will always be set to a value of 0. If creating multiple filters, pass in values using comma separator. Supported', required=False)
+parser.add_argument('--value', help='Create OME query group filter, pass in suported value for the field ID you are selecting. Supported values are either integer, string or set value from a list. If creating multiple filters, pass in value using comma separator', required=False)
+parser.add_argument('-q', help='Get query information for OME group, pass in the group ID. If needed, execute argument -g to get the group ID.', required=False)
+parser.add_argument('-d', help='Get devices detected by OME group, pass in the group ID. If needed, execute argument -g to get the group ID.', required=False)
+parser.add_argument('-D', help='Delete OME group, pass in the group ID. If needed, execute argument -g to get the group ID.', required=False)
+
+
+args = vars(parser.parse_args())
+
+ome_ip=args["ip"]
+ome_username=args["u"]
+ome_password=args["p"]
+
+
+
+def get_oid_fid_value_possible_values():
+    print("\n- WARNING, collecting OME query group operator and field ID information for OME %s\n" % ome_ip)
+    try:
+        os.remove("create_OME_query_group_paramater_help.txt")
+    except:
+        pass
+    file_open = open("create_OME_query_group_paramater_help.txt", "a")
```

Comment:
I think this is cleaner using a with statement - with  open("file","a") as file_open:  There shouldn't be a need for an explicit close later